lib/qrda-export/helper/aggregate_object_helper.rb

```diff
@@ -98,30 +98,27 @@ def is_cv?
       class AggregateCount
         attr_accessor :measure_id,  :populations, :population_groups
 
+        ALL_POPULATION_CODES = %w[IPP DENOM NUMER NUMEX DENEX DENEXCEP MSRPOPL MSRPOPLEX OBSERV].freeze
```

Comment:
We use these in Cypress too, in at least one place. Does it make sense to move these into a shared library? Maybe a constant on `Measure`?

Reply:
I believe they are already on `Measure` https://github.com/projecttacoma/cqm-models/blob/40b73f19f769415ccfd81c5161b856b64a3e3081/app/models/cqm/measure.rb#L25

Reply:
Yeah, it has STRAT removed...since it isn't really a population.

Reply:
I'm not sure how I feel about having the same constant name, with different values. Is there a reason for STRAT being in the `Measure` constant/should we remove it there?

Reply:
This is no longer specified as a constant.